requirements/common.txt

```diff
@@ -108,19 +108,20 @@ mozlog==3.5 \
     --hash=sha256:30fa398c631faf2f6b8cc81d95e35e3ee9f1a6e5bcdf3132fde7ad943273d66e \
     --hash=sha256:2b70acdbdd4385a26083b298ba86f30fce82d5a0e662a07025a2f17553be4fa1
 
-futures==3.1.1 \
+futures==3.1.1; python_version < '3' \
     --hash=sha256:c4884a65654a7c45435063e14ae85280eb1f111d94e542396717ba9828c4337f \
     --hash=sha256:51ecb45f0add83c806c68e4b06106f90db260585b25ef2abfcda0bd95c0132fd
 
 # Required by jsonschema
-functools32==3.2.3-2 --hash=sha256:89d824aa6c358c421a234d7f9ee0bd75933a67c29588ce50aaa3acdf4d403fa0
+functools32==3.2.3-2; python_version < '3' \
+    --hash=sha256:89d824aa6c358c421a234d7f9ee0bd75933a67c29588ce50aaa3acdf4d403fa0
 
 # Required by django-rest-swagger
 coreapi==2.3.1 \
     --hash=sha256:af112b32104d0dbdedbac8e4c22d6068b799efad0c2328204575aad66de8f761 \
     --hash=sha256:af88b5ad7415410de9ef33bd7f7cf605534f1c4cf24de716ebe4453021249714
 coreschema==0.0.4 \
-    --hash=sha256:5e6ef7bf38c1525d5e55a895934ab4273548629f16aed5c0a6caa74ebf45551f
+    --hash=sha256:9503506007d482ab0867ba14724b93c18a33b22b6d19fb419ef2d239dd4a1607
```

Comment:
For this and `mohawk`, could you add the new hash in addition to the old one, rather than replacing it? It's just that the old hash is still needed for Python 2, and otherwise clean installs for Python 2 will start failing. (The reason failures aren't seen on this PR's Travis run, is that the virtualenv is cached from job to job, to reduce the setup time)

To add multiple hashes for one entry, use the line continuation symbol, ` \` - there are a few examples just above this line in the file :-)